api/room/schema.py

```diff
@@ -54,12 +55,29 @@ def mutate(self, info, room_id, **kwargs):
         return UpdateRoom(room=exact_room)
 
 
+class DeleteRoom(graphene.Mutation):
+
+    class Arguments:
+        room_id = graphene.Int(required=True)
+    room = graphene.Field(Room)
+
+    def mutate(self, info, room_id, **kwargs):
+        query_room = Room.get_query(info)
+        exact_room = query_room.filter(
+            RoomModel.id == room_id).first()
+        if not exact_room:
+            raise GraphQLError("RoomId not found")
+
+        exact_room.delete()
+        return DeleteRoom(room=exact_room)
+
+
 class Query(graphene.ObjectType):
     all_rooms = graphene.List(Room)
     get_room_by_id = graphene.Field(
         Room,
         room_id=graphene.Int()
-        )
+    )
     room_schedule = graphene.Field(
         Calendar,
         calendar_id=graphene.String(),
```

Comment:
I see that this room_schedule field is declared twice on your branch. Please check and remove one of the fields.

Reply:
@PatrickCmd ,Thanks for correction.All is well now